src/ui/screens/DemocracyScreen.tsx

```diff
@@ -65,6 +75,7 @@ export function DemocracyScreen() {
                 </>
               );
             }}
+            renderSectionFooter={groupedDataEmpty}
```

Comment:
NIT 💅 

If the logic is implemented directly here we don't have to create extra types in the function above.
Also we can drop the `<Card />` here.

```suggestion
            renderSectionFooter={({section}) =>
              section.data.length < 1 ? (
                <View style={globalStyles.paddedContainer}>
                  <EmptyState subheading="There are no items to display" caption="Please check back for updates" />
                </View>
              ) : null
            }
```

Reply:
We need to have the `Card`, else the  whole background will be white, ref :

<img src="https://user-images.githubusercontent.com/22363352/159910361-6c9c9221-e3fc-43c0-8de9-4ac35520f4a1.png" width="250" />